samples/bluetooth/fast_pair/locator_tag/src/fp_adv.c

```diff
@@ -30,6 +30,11 @@ LOG_MODULE_DECLARE(fp_fmdn, LOG_LEVEL_INF);
 #define FP_RPA_TIMEOUT			(13 * FP_RPA_SEC_PER_MIN)
 #define FP_RPA_TIMEOUT_OFFSET_MAX	(2 * FP_RPA_SEC_PER_MIN)
 
+/* Max number of triggers that can be registered with app_fp_adv_trigger_register()
+ * to later use the app_fp_adv_request().
+ */
+#define MAX_ADV_TRIGGERS (4)
```

Comment:
Using code section and macro to register an element there could allow us to avoid configuration misalignment possibility. I would also simplify implementation and speed up execution (no need to look through array element by element; can rely on pointer difference)